packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -243,8 +243,8 @@ export const messages = {
   force_apex_log_get_no_logs_text: 'No Apex debug logs were found',
   force_apex_log_get_pick_log_text: 'Pick an Apex debug log to get',
   force_apex_log_list_text: 'Getting Apex debug logs',
-  force_apex_log_get_library: 'Apex Library: Get Logs',
   apex_log_get_text: 'Get Apex Logs',
+  apex_log_list_text: 'Fetching Apex debug log list',
```

Comment:
@ekapner let me know what you think of this text! This will be displayed when the user is waiting for a list of their debug logs to be displayed

Reply:
@AnanyaJha - yeah, I think we've used "fetching" before - so that would be consistent (if I'm remembering correctly). So, this looks good.